setup.py

```diff
@@ -90,15 +85,13 @@
     "SQLAlchemy~=1.3.0",
     "sklearn-crfsuite~=0.3.6",
     "PyJWT~=1.7",
-    # remove when tensorflow@1.15.x or a pre-release patch is released
-    # https://github.com/tensorflow/tensorflow/issues/32319
-    "gast==0.2.2",
+    "transformers~=2.3.0",
 ]
 
 extras_requires = {
     "test": tests_requires,
     "spacy": ["spacy>=2.1,<2.2"],
-    "convert": ["tensorflow_text~=1.15.1", "tensorflow_hub~=0.6.0"],
+    "convert": ["tensorflow_text~=2.1.0rc0", "tensorflow_hub~=0.7.0"],
```

Comment:
can we use `2.1.1`?

Reply:
@dakshvar22

Reply:
updated the dependency rule to `>=2.1.0rc0`. So it brings in `2.1.1` if possible. `2.1.1` is not compatible with `MacOS<10.15`